legend-engine-xt-sql-postgres-server/src/main/java/org/finos/legend/engine/postgres/PostgresServer.java

```diff
@@ -132,12 +121,41 @@ public NioEventLoopGroup getWorkerGroup()
     public static void main(String[] args)
     {
         int port = 9998;
+        String protocol;
```

Comment:
Given we are starting to have more configs, can we add config management for the db server?

Reply:
done